### PR TITLE
Add support for storage endpoints.

### PR DESCRIPTION
**Pull request by phillbaker**
Addresses https://github.com/digitalocean/droplet_kit/issues/87.

```
config = DropletKit::Volume.new(
  size_gigabytes: 10,
  name: "Example",
  description: "Block store for examples",
  region: "nyc1"
)
drive = client.volumes.create(drive)
```

block storage is in beta and thus the API will change. We'll keep this code in a branch until we find a way to include this in the client in a way that will not break users if they come to rely on the changes here.

cc @mjsteger

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -272,6 +272,27 @@ Actions supported:
 * `client.floating_ip_actions.assign(ip: floating_ip.ip, droplet_id: droplet.id)`
 * `client.floating_ip_actions.unassign(ip: floating_ip.ip)`
 
+## Drive resource
```
Comment (nickvanw) [inline README.md]: They're called volumes

File: lib/droplet_kit/mappings/droplet_mapping.rb [hunk 2]
```diff
@@ -29,6 +29,7 @@ class DropletMapping
       # Create properties arent quite the same
       property :name, scopes: [:create]  # "Regular" create
       property :names, scopes: [:create] # Multiple create
+      property :drives, scopes: [:create] # Create with drives
```
Comment (nickvanw) [inline lib/droplet_kit/mappings/droplet_mapping.rb]: `property :volumes`

File: lib/droplet_kit/models/droplet.rb [hunk 3]
```diff
@@ -8,6 +8,7 @@ class Droplet < BaseModel
 
     # Used for creates
     attribute :names
+    attribute :drives
```
Comment (nickvanw) [inline lib/droplet_kit/models/droplet.rb]: `attribute :volumes`

**Comment by phillbaker** [comment]
Updated, thanks.

File: README.md [hunk 4]
```diff
@@ -272,6 +272,27 @@ Actions supported:
...
+    client.volumes #=> DropletKit::VolumeResource
+
+Actions supported:
+
+* `client.volumes.all()`
+* `client.volumes.find(id: 'id')`
+* `client.volumes.create(volume)`
+* `client.volumes.delete(id: 'id')`
+
+## Floating IP Action resource
```
Comment (nickvanw) [inline README.md]: `Volume Action resource`

Comment (phillbaker) [inline README.md]: 🐼

**Comment by nickvanw** [comment]
LGTM

**Pull request merged**